### PR TITLE
Adds a few new emotes

### PR DESCRIPTION
**Pull request by etherware-novice**
## About The Pull Request
Adds 3 new emotes I felt were needed for good roleplaying (thumbs up, down, and whistle)

## Why It's Good For The Game
immersion

## Testing Photographs and Procedure

emotes

<details>
<summary>Screenshots&Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

## Changelog
:cl:
add: Added the 3 new emotes
/:cl:

### CODE REVIEW

**Comment by Fronsis** [comment]
Do we have already have the one where you can *slap a table? [As seen here time stamped in the video](https://youtu.be/zVyvUF25pgg?t=1215) Or the one where you crack your knuckles! those two are very good imo

**Comment by etherware-novice** [comment]
i could add them to this if they arent already in? i just added a few i would like

**Review by PowerfulBacon — CHANGES_REQUESTED** [review]
PR body is incomplete. All sections must be filled out with appropriate information. Testing evidence is required even for small PRs.

**Comment by etherware-novice** [comment]
what should i put for testing evidence?

**Comment by PowerfulBacon** [comment]
Just a screenshot of you using them and the output in the chat.

**Comment by LodedDiper69** [comment]
You should let people not be able to thumbs up when they have no arms

**Review by PowerfulBacon — DISMISSED** [review]

File: code/modules/mob/living/emote.dm [hunk 1]
```diff
@@ -595,3 +595,34 @@
...
+/datum/emote/living/whistle
+	key="whistle"
+	key_third_person="whistle"
+	message = "whistles a few notes"
+	message_robot = "whistles a few synthesized notes"
+	message_AI = "whistles a synthesized song"
+	message_ipc = "whistles a few synthesized notes"
+
+/datum/emote/living/whistle/get_sound(mob/living/user)
+	return 'sound/items/megaphone.ogg'
```
Comment (PowerfulBacon) [inline code/modules/mob/living/emote.dm:630]: This probably will sound very strange in game, since its an electronic sound rather than a whistle sound.

**Comment by etherware-novice** [comment]
> You should let people not be able to thumbs up when they have no arms

hhow do i do that?

**Comment by PowerfulBacon** [comment]
Not sure, there should be an emote saying can use. Check high five or another arm emote like salute

**Review by PowerfulBacon — APPROVED** [review]

**Pull request merged**